Src/Graphics/Camera.h

```diff
@@ -15,7 +15,7 @@ class Camera {
 
         float yawAngle;
         float pitchAngle;
-        // The range of the the pitchAngle field before it is clamped.
+        // The range of the the pitchAngle field before it is clamped. - Gimbal lock workaround
```

Comment:
Not a gimbal lock workaround. This is simply because the camera in the game is deliberately clamped to a max pitch of 70 degrees, which we may or may not want to tweak. Gimbal lock is not a concern with the player's camera.

Reply:
I think putting the camera upside-down happens when the player dies and looks straight up.
It may be appropriate to clamp this by other means.
Not sure.

Reply:
Feel free to do some experimentation, but be aware that much of camera and player movement should be dealt with in AngelScript, which probably requires exposing more functionality to it.